drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/HtmlUnitDriverFactory.java

```diff
@@ -86,4 +101,22 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
     protected String getDriverReadableName() {
         return BROWSER_CAPABILITIES;
     }
+
+    private void setClientOptions(WebClient webClient, String htmlUnitWebClientOptions) {
+        final WebClientOptions webClientOptions = webClient.getOptions();
+
+        Map<String, String> clientOptions = new LinkedHashMap<>();
+        final String[] options = htmlUnitWebClientOptions.split(",");
```

Comment:
how about using semicolon to delimit key-values and coma for args:
~~~xml
<property name=hmtlUnitWebClientOptions>timeout=10;SSLClientProtocols=foo,bar,test</property> 
~~~
from my point of view, it looks more natural